tests/test_issues/input/Issue_6_fixed.yaml

```diff
@@ -7,11 +7,14 @@ prefixes:
 emit_prefixes:
   - dct
 
-imports: [metat:types]
+imports:
+  - metat:types
 default_range: string
 
 slots:
-  created: {range: datetime, slot_uri: dct:created}  # slot_uri: "dct:created"
+  created:
+    range: datetime
+    slot_uri: dct:created  # slot_uri: "dct:created"
```

Comment:
I would just remove the commented-out part unless there is a very compelling reason to keep it.